webservices/resources/candidates.py

```diff
@@ -84,7 +98,9 @@ class CandidateSearch(CandidateList):
 
     schema = schemas.CandidateSearchSchema
     page_schema = schemas.CandidateSearchPageSchema
-    query_options = [sa.orm.subqueryload(models.Candidate.principal_committees)]
+    query_options = [
+        sa.orm.subqueryload(models.Candidate.principal_committees),
```

Comment:
@jmcarp:  You're absolutely right, I'm sorry.  I mixed up two separate issues we ran into and confused the two, and in my haste attributed the problem incorrectly.  Please disregard the note above, the issues is with the way the joins themselves are constructed and the SQL that's ultimately generated, **not** `nplusone`!